onnxruntime/core/providers/dml/OperatorAuthorHelper/MLOperatorAuthorPrivate.h

```diff
@@ -6,16 +6,30 @@
 interface IDMLOperation;
 interface IDMLOperator;
 struct DML_OPERATOR_DESC;
-
-struct MLOperatorKernelDmlProperties
+struct DML_INPUT_GRAPH_EDGE_DESC;
+struct DML_OUTPUT_GRAPH_EDGE_DESC;
+struct DML_INTERMEDIATE_GRAPH_EDGE_DESC;
+
+// Either nodesAsOpDesc or nodesAsIDMLOperator be present
+//  1) Operator kernels which implements operator using only DML Operator, they will pass DML_OPERATOR_DESC. 
```

Comment:
>operator using only DML Operator, they will pass DML_OPERATOR_DESC.  [](http://example.com/codeflow?start=41&length=68)

How about..

"...implement that operator using only a single DML operator will pass a DML_OPERATOR_DESC."